testing/Makefile.am

```diff
@@ -47,6 +47,7 @@ EXTRA_DIST			= ibft xpath \
 
 TESTS				= socket-mock-test	\
 				  bitmask-test		\
-				  bitmap-test
+				  bitmap-test		\
+				  test-nbft
```

Comment:
Why to follow conventions -- everything else is using `*-test` -- why nbft `test-nbft`?
Please also make sure that `make distcheck` works.

Reply:
* renamed it to `nbft-test.sh`
 * fixed `make distcheck` thx @mwilck

Reply:
OK, `./autogen.sh && make rpmpackage` works now:
```
make[2]: Verzeichnis „/space/mt/work/wicked/git/testing“ wird betreten
make[3]: Verzeichnis „/space/mt/work/wicked/git/testing“ wird betreten
PASS: socket-mock-test
PASS: bitmask-test
PASS: bitmap-test
PASS: nbft-test.sh
============================================================================
Testsuite summary for wicked 0.6.71
============================================================================
# TOTAL: 4
# PASS:  4
# SKIP:  0
# XFAIL: 0
# FAIL:  0
# XPASS: 0
# ERROR: 0
============================================================================
make[3]: Verzeichnis „/space/mt/work/wicked/git/testing“ wird verlassen
```